src/Hl7.Fhir.Specification.STU3.Tests/Snapshot/SnapshotGeneratorTest.cs

```diff
@@ -2433,8 +2433,8 @@ private static bool isAlmostExactly(ElementDefinition elem, ElementDefinition ba
             }
 
             // Also ignore any Changed extensions on base and diff
-            elemClone.RemoveAllConstrainedByDiffExtensions();
-            baseClone.RemoveAllConstrainedByDiffExtensions();
+            elemClone.RemoveAllNonInheritableExtensions();
```

Comment:
Does this method replace the original ones? So RemoveallNonInheritableExtension replaces RemoveAllConstrainedByDiffExtensions?  If so, should we not remove/obsolete the original method?

Reply:
Well, I thought about that, but didn't dare to since Forge might be using the ConstraintByDiff extension one maybe?
And that is a bit faster (I don't know if it might be noticeable in a UI).

Reply:
Thanks, I'll Obsolete is then

Reply:
I'm working on supporting extensions on element definnition properties in Forge. That would mean the UI won't be cluttered with these I guess? That would be nice.
![image](https://github.com/user-attachments/assets/ed4c6a6b-2a48-4499-bab0-642dc5b2307c)

Reply:
How about http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName? I see that one all thhe time as well.

Reply:
This is not on the list:
We only made this fix for category 5 of this list:
https://jira.hl7.org/browse/FHIR-28441?focusedCommentId=177188&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-177188